docs/5.0/preview/database-access.md

```diff
@@ -38,6 +38,134 @@ With Database Access users can:
 Your browser does not support the video tag.
 </video>
 
+## Getting Started
+
+In this guide we will use Teleport Database Access to connect to a PostgreSQL
+flavored AWS Aurora database. Here's an overview of what we will do:
+
+1. Configure AWS Aurora database with IAM authentication.
+2. Download and install Teleport and connect it to the Aurora database.
+3. Connect to the Aurora database via Teleport.
+
+### Step 1/3. Setup Aurora
+
+In order to allow Teleport connections to an Aurora instance, it needs to support
+IAM authentication.
+
+If you don't have a database provisioned yet, create an instance of an Aurora
+PostgreSQL instance in the [RDS control panel](https://console.aws.amazon.com/rds/home).
```

Comment:
For a fresh DB you need to select `Standard create` to enable Password and IAM

Reply:
I've clarified this below.